x-pack/legacy/plugins/ml/public/data_frame_analytics/common/fields.ts

```diff
@@ -68,6 +70,104 @@ export const sortColumns = (obj: EsDocSource, resultsField: string) => (a: strin
   return a.localeCompare(b);
 };
 
+export const sortRegressionResultsFields = (
+  a: string,
+  b: string,
+  jobConfig: DataFrameAnalyticsConfig
+) => {
+  const dependentVariable = getDependentVar(jobConfig.analysis);
+  const resultsField = jobConfig.dest.results_field;
+  const predictedField = getPredictedFieldName(resultsField, jobConfig.analysis);
+  if (a === `${resultsField}.is_training`) {
+    return -1;
+  }
+  if (b === `${resultsField}.is_training`) {
+    return 1;
+  }
+  if (a === predictedField) {
+    return -1;
+  }
+  if (b === predictedField) {
+    return 1;
+  }
+  if (a === dependentVariable) {
+    return -1;
+  }
+  if (b === dependentVariable) {
+    return 1;
+  }
+  return a.localeCompare(b);
+};
+
+// Used to sort columns:
+// Anchor on the left ml.is_training, <predictedField>, <actual>
+export const sortRegressionResultsColumns = (
```

Comment:
Is the predicted field getting right alignment because it's a nested field? Wonder if it would look better left aligned like the other numeric columns? Otherwise you end up with a big gap between the first and second columns:

![image](https://user-images.githubusercontent.com/7405507/67941848-72d66a00-fbce-11e9-9544-84182788da6f.png)